### ISSUE TITLE
treating a potential boolean as an array.

### ISSUE THREAD

**Issue by proseLA** [torvista/Zen_Cart-Google_Merchant_Center_Feeder#7]
i spoke too soon!

`$raw` can return false.  see:

https://github.com/torvista/Zen_Cart-Google_Merchant_Center_Feeder/blob/8c7b87ed77a9e2573dfe81d64b88c629581da368/catalog/includes/classes/google_base.php#L1063-L1064

later on, you are treating it as an array.

https://github.com/torvista/Zen_Cart-Google_Merchant_Center_Feeder/blob/8c7b87ed77a9e2573dfe81d64b88c629581da368/catalog/includes/classes/google_base.php#L1080

that's a problem.

it is generating logs for me.

**Comment by torvista** [comment]
Yes, I see that, feel free to butcher/PR as you see fit, I don't use this mod at the moment and am too dizzy with other issues to do it myself.
